PWGHF/Tasks/taskXicc.cxx

```diff
@@ -48,16 +49,16 @@ struct HfTaskXicc {
   void init(o2::framework::InitContext&)
   {
     auto vbins = (std::vector<double>)bins;
-    registry.add("hmass", "#Xi^{++}_{cc} candidates;inv. mass (p K #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
+    registry.add("hmass", "#Xi^{++}_{cc} candidates;inv. mass (p K #pi #pi) (GeV/#it{c}^{2});entries", {HistType::kTH2F, {{400, 3.2, 4.0}, {vbins, "#it{p}_{T} (GeV/#it{c})"}}});
```

Comment:
Can you check whether these histograms have really `"#it{p}_{T} (GeV/#it{c})"` on the y axis and not `"entries"`?

Reply:
The output histogram has a name `#it{p}_{T} (GeV/#it{c})` on the y axis.